app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -841,25 +868,46 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     }
 
     private fun setUpUiStateObserver() {
-        lifecycleScope.launchWhenStarted {
-            viewModel.uiState.collect { uiState ->
-                uiState.uiMessages.firstOrNull()?.let {
-                    Toast.makeText(this@ConversationActivityV2, it.message, Toast.LENGTH_LONG).show()
-                    viewModel.messageShown(it.id)
-                }
-                if (uiState.isMessageRequestAccepted == true) {
-                    binding.messageRequestBar.visibility = View.GONE
```

Comment:
Is the banner still hidden when accepted? I don't see the new version of this code

Reply:
Ok I saw some code for this further down

Reply:
There's something missing here and it has been corrected